### DISCUSSION TITLE
Organize Form fields to skip column

### DISCUSSION THREAD

**Discussion by pedrocoelho98** [filamentphp/filament#3169] [Help] [answered]
Hi.

Let's say we have a 6 column form.

Let's make the "address" field columnSpan(4) for example.

The next fields are "ZIP Code" and "City".

How would you make the "ZIP code" skip the 5 and 6 column and forcing them to go to the next row and keep them together?

PS- I'm quit new to filament and php, so keep that in mind!

Thanks

**Comment by danharrin** [comment, marked as answer]
Grid with 3 columns
    - Address field with `columnSpan(2)`
Grid with 3 columns
    - Zip code field
    - City field

This is the easiest way. Otherwise you have to build a custom theme to get extra Tailwind classes
